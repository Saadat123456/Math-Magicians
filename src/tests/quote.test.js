import React from 'react';
import renderer from 'react-test-renderer';
import Quote from '../pages/quote';

it('renders correctly', () => {
  const tree = renderer
    .create(<Quote ></Quote>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});