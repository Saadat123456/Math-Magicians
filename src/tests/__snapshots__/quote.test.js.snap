// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<main
  className="quote mainStyle"
>
  <div
    className="details"
  >
    <h2
      className="heading"
    >
      
    </h2>
    <div
      className="description descripition-quote"
    >
      <blockquote
        cite="http://www.worldwildlife.org/who/index.html"
      >
        <span
          className="quotes"
        >
          “
        </span>
        It is better to solve one problem five different ways than to solve five problems one way.
        <span
          className="quotes"
        >
          ”
        </span>
      </blockquote>
    </div>
  </div>
  <span />
</main>
`;
